beacon-chain/sync/pending_attestations_queue.go

```diff
@@ -20,14 +21,20 @@ import (
 )
 
 // This defines how often a node cleans up and processes pending attestations in the queue.
-var processPendingAttsPeriod = slots.DivideSlotBy(2 /* twice per slot */)
+
 var pendingAttsLimit = 10000
 
 // This processes pending attestation queues on every `processPendingAttsPeriod`.
 func (s *Service) processPendingAttsQueue() {
+	clock, err := s.clockWaiter.WaitForClock(s.ctx)
+	if err != nil {
+		log.WithError(err).Error("attestation queue failed to receive genesis data")
+		return
+	}
 	// Prevents multiple queue processing goroutines (invoked by RunEvery) from contending for data.
 	mutex := new(sync.Mutex)
-	async.RunEvery(s.ctx, processPendingAttsPeriod, func() {
+	processPendingAttsPeriod := slots.DivideSlotBy(2 /* twice per slot */)
+	async.RunWithTickerAndInterval(s.ctx, clock.GenesisTime(), []time.Duration{0, processPendingAttsPeriod}, func() {
```

Comment:
I think we should avoid triggering go routines at second 0. I propose we process pending attestations at 5 seconds and 11 seconds.  

I think that the right approach here is what's done in this PR instead of tickers https://github.com/prysmaticlabs/prysm/pull/9679

Pending atts should be processed immediately after processing the block on demand, there's no use on processing them at any other time. It is also simpler to implement: just spin up a routine after `postProcessBlock`.